docs/proposals/20220825-azuremanaged-cluster-v1.md

```diff
@@ -0,0 +1,67 @@
+---
+title: AzureManagedCluster graduation to v1 from experimental
+authors:
+  - "@jackfrancis"
+reviewers:
+  - @CecileRobertMichon
+  - @zmalik
+  - @NovemberZulu
+creation-date: 2022-08-25
+last-updated: 2022-08-25
+status: implementable
+see-also:
+- https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/2204
+- https://github.com/kubernetes-sigs/cluster-api/pull/6988
+---
+
+
+# AzureManagedCluster graduation to v1 from experimental
+
+## Summary
+
+`AzureManagedCluster` is a capz-native implementation of Azure Managed Kubernetes (AKS). Because there is no standard set of Cluster API resource definitions for a "Managed Kubernetes cluster", it is left up to the provider to re-use the existing Cluster API specification (for example, the `Cluster` and its to-be-implemented-by-provider properties such as `ControlPlaneEndpoint`, `ControlPlaneRef` and `InfrastructureRef`). As a result, capz implemented `AzureManagedCluster` with an API contract designation of "experimental", to allow for rapid prototyping and discovery.
+
+With the recent adoption of `AzureManagedCluster` by the capz community for practical, real-world use, we want to identify the set of outstanding items that may prevent graduation to v1, and address each one of them, so that future adoption can be unlocked, and users can confidently build resilient systems on top of a stable API.
+
+## Motivation
+
+### Goals
+- Agree upon a definitive v1 specification of `AzureManagedCluster`
```

Comment:
I imagine this would also include associated types like AzureManagedControlPlane and AzureManagedMachinePool?

Reply:
added this mention